### PR TITLE
evaluation result list page

### PR DESCRIPTION
**Pull request by yuki-mt**
**I will remove WIP after #103 is merged** 

## What is this PR for?

add page for EvaluationResult list

### frontend
- add link to SideMenu
- show EvaluationResult list
- add `Evaluate model` button
- able to delete EvaluationResult with checkbox
### backend
- add API to return list of EvalualtionResult

## What type of PR is it?

Feature

## What is the issue?

#23 

## How should this be tested?

`python -m unittest test/apis/test_api_evaluation.py`

### CODE REVIEW

**Comment by yuki-mt** [comment]
I merged test fix from #109 . please review it now @keigohtr

File: rekcurd_dashboard/apis/api_evaluation.py [hunk 1]
```diff
@@ -195,6 +195,7 @@ def post(self, project_id: int, application_id: str):
         args = self.eval_parser.parse_args()
         eval_id = args.get('evaluation_id', None)
         model_id = args.get('model_id')
+        do_overwrite = args.get('overwrite')
```
Comment (keigohtr) [inline rekcurd_dashboard/apis/api_evaluation.py]: At the moment, it is OK.
Hopefully, we should use `PATCH` or `PUT` method for this.

File: rekcurd_dashboard/apis/api_evaluation.py [hunk 2]
```diff
@@ -245,9 +248,25 @@ def post(self, project_id: int, application_id: str):
...
+            result = row.EvaluationResultModel.serialize
+            result['model'] = row.ModelModel.serialize
+            result['evaluation'] = row.EvaluationModel.serialize
+            results.append(result)
+        return results
+
 
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results/<int:eval_result_id>')
+class ApiEvaluationResultId(Resource):
     def get(self, project_id: int, application_id: str, eval_result_id: int):
```
Comment (keigohtr) [inline rekcurd_dashboard/apis/api_evaluation.py]: ditto

File: rekcurd_dashboard/apis/api_evaluation.py [hunk 3]
```diff
@@ -245,9 +248,25 @@ def post(self, project_id: int, application_id: str):
         return response_body
 
 
-@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results/<int:eval_result_id>')
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results')
 class ApiEvaluationResults(Resource):
+    def get(self, project_id: int, application_id: str):
```
Comment (keigohtr) [inline rekcurd_dashboard/apis/api_evaluation.py]: Could you add a serializer?
```
@evaluation_api_namespace.expect(hoge)
@evaluation_api_namespace.marshal_with(fuga)
```

File: rekcurd_dashboard/apis/api_evaluation.py (hunk 1 above)
Comment (yuki-mt) [reply rekcurd_dashboard/apis/api_evaluation.py]: oh, that's right. I will fix it

Comment (yuki-mt) [reply rekcurd_dashboard/apis/api_evaluation.py]: @keigohtr I removed `overwrite` parameter and use `PUT` to re-evalualte model
backend: https://github.com/rekcurd/dashboard/pull/105/commits/0fb0dd67c39be7ccc0996c1aa50c761fa84940db
frontend: https://github.com/rekcurd/dashboard/pull/105/commits/fe95fcb06eb3e3cbb78c83f200115382ff2d3595

File: rekcurd_dashboard/apis/api_evaluation.py (hunk 3 above)
Comment (yuki-mt) [reply rekcurd_dashboard/apis/api_evaluation.py]: fixed in https://github.com/rekcurd/dashboard/pull/105/commits/e0a21c7d65247b96070541c49a52202fbef180cc

File: rekcurd_dashboard/apis/api_evaluation.py (hunk 2 above)
Comment (yuki-mt) [reply rekcurd_dashboard/apis/api_evaluation.py]: fixed in https://github.com/rekcurd/dashboard/pull/105/commits/e0a21c7d65247b96070541c49a52202fbef180cc

**Comment by yuki-mt** [comment]
@keigohtr I just fixed the `Invalid Date` in EvaluationResult!

**Pull request merged**